docs/pages/product/configuration/data-sources/materialize.mdx

```diff
@@ -29,6 +29,7 @@ CUBEJS_DB_PORT=6875
 CUBEJS_DB_NAME=materialize
 CUBEJS_DB_USER=materialize
 CUBEJS_DB_PASS=materialize
+CUBEJS_MZ_CLUSTER=quickstart
```

Comment:
Similarly to other DB-specific env vars ([example](https://cube.dev/docs/reference/configuration/environment-variables#cubejs_db_duckdb_memory_limit)), let's call this one `CUBEJS_DB_MATERIALIZE_CLUSTER`.

Reply:
Just updated the env var to `CUBEJS_DB_MATERIALIZE_CLUSTER`